examples/node/instant-buy.js

```diff
@@ -0,0 +1,88 @@
+import { OrditApi, Ordit } from '@sadoprotocol/ordit-sdk'
+
+const BUYER_MNEMONIC = `<12-WORDS-PHRASE>`
```

Comment:
Why backticks?

Reply:
Since this is not a template literal, I would suggest using regular double quotes instead for consistency.